docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md

```diff
@@ -0,0 +1,17 @@
+# SGX1 mode: Prereqs
+
+## Intel SGX Platform Software for Windows (PSW) v2.2
+
+The PSW should be installed automatically on Windows 10 with the Fall Creators
+Update installed, or on a Windows Server 2016 image for an Azure Confidential
+Compute VM. You can verify that is the case on the command line as follows:
+
+```cmd
+sc query aesmservice
+```
+
+The state of the service should be "running" (4). Follow Intel's documentation for troubleshooting.
+
+TODO: Provide link for donwloading PSW 2.2
+
+Note that Open Enclave is only compatible with the Intel PSW 2.2.
```

Comment:
This is not quite true. It's more accurate to say that:

> Note that Open Enclave running on systems that only support SGX1 without Flexible Launch Control (FLC) is only compatible with Intel SGX PSW 2.2. To use newer versions of the Intel SGX PSW, you need to [install on a system that supports FLC](WindowsManualSGX1FLCDCAPPrereqs.md).

Reply:
Removed SGX1